R/conjugate_encoder.R

```diff
@@ -10,7 +10,7 @@ library(rlang)
 #' per each category are used as the encodings.
 #'
 #' @param X_train A tibble representing the training data set containing some categorical features/columns.
-#' @param X_test A tibble representing the test set, containing some set of categorical features/columns.
+#' @param X_test An optional tibble representing the test set, containing some set of categorical features/columns. Default = NA.
```

Comment:
Same difference of NA vs NULL, changing for consistency
```suggestion
#' @param X_test An optional tibble representing the test set, containing some set of categorical features/columns. Default = NULL.
```